cpp/include/typedb/query/querymanager.hpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#pragma once
+
+#include "typedb/common/native.hpp"
+#include "typedb/common/future.hpp"
+#include "typedb/connection/options.hpp"
+
+#include "typedb/concept/concept.hpp"
+#include "typedb/answer/conceptmap.hpp"
+#include "typedb/answer/conceptmapgroup.hpp"
+#include "typedb/answer/valuegroup.hpp"
+#include "typedb/answer/valuefuture.hpp"
+
+#include "typedb/logic/explainable.hpp"
+#include "typedb/logic/explanation.hpp"
+
+namespace TypeDB {
+
+typedef std::string JSONString;
+using JSONIterable = TypeDB::StringIterable;
```

Comment:
TODO: Implement the minimal JSON

Do we want to return a custom "JSONObject" or just a string and give them a `JSONObject.from(std::string&)`  method in case they want to use ours?